shared/cfg/base.cfg

```diff
@@ -699,7 +699,7 @@ Linux:
     stress_install_from_repo = "no"
     # param for stress tests
     stress_args = '--cpu 4 --io 4 --vm 2 --vm-bytes 256M'
-    download_url_stress = 'http://people.seas.harvard.edu/~apw/stress/stress-1.0.4.tar.gz'
+    download_url_stress = '/home/kar/workspace/avocado-vt/shared/deps/stress/stress-1.0.4.tar.gz'
```

Comment:
If you want to use a local source package, `data_dir.get_deps_dir()` is more flexible than an absolute path, and I think if it only works for this test case, it is best to move it to `deps_dir` of tp-qemu. Otherwise, @balamuruhans has provided a new url link to replace, see the comments above.

Reply:
It is generic for tp-libvirt and tp-qemu test cases. I believe moving it to deps_dir of tp-qemu is not suggested.

Reply:
I searched the code for a bit, maybe `utils_misc.get_path(data_dir.DEPS_DIR, stress-1.0.4.tar.gz)` is right for us. We can save this package to `avocado-vt/shared/deps/stress/stress-1.0.4.tar.gz`
And you're right, tp-libvirt has one line about download_url_stress: https://github.com/autotest/tp-libvirt/blob/a7c4001e748f63fe6dfb45aaf5d6d624d25500d9/libvirt/tests/cfg/virsh_cmd/domain/virsh_migrate_stress.cfg#L54
So it also needs to be modified.

Reply:
cool